third_party/icu/googleurl/BUILD

```diff
@@ -0,0 +1,27 @@
+load(
```

Comment:
Not so sure about the this new tree, `third_party`. It makes sense to have something like this, but I kind of feel that we also have some of what belongs here under `bazel/` today. Is there some overall plan here for using this tree?

Reply:
My reasoning will be: there is no *.h and *.cc inside `bazel/` today, but we have `third_party/statusor` which has `*.h` and `*.cc`. I'm OK moving this under `bazel/` but I think it is better to have it grouped in a dir (BUILD and source codes).

Reply:
Oops. I didn't see this :(. I have moved it under  `bazel/` per your suggestion.